android/src/main/kotlin/io/bucketeer/sdk/flutter/BucketeerFlutterClientSdkPlugin.kt

```diff
@@ -383,16 +408,20 @@ class BucketeerFlutterClientSdkPlugin : MethodCallHandler, FlutterPlugin {
         }
 
       }
-    } catch (e: BKTException) {
-      fail(result, e.message)
-    } catch (e: RuntimeException) {
-      fail(result, e.message)
+    } catch (e: Exception) {
+      fail(result, e.message, exception = e)
     }
   }
 
-  @Throws(IllegalStateException::class)
+  @Throws(Exception::class)
   private fun assertInitialize() {
-    BKTClient.getInstance()
+    try {
+      BKTClient.getInstance()
+    } catch (ex: BKTException.IllegalStateException) {
+      throw  ex
```

Comment:
correct this to throw BKTException.IllegalStateException so we will not need to wait for new version 2.0.6 of the Android SDK.